pkg/daemon/manager/manager.go

```diff
@@ -109,6 +146,55 @@ func (m *manager) EnsureServiceState(service winsvc.Service, state svc.State) er
 	return winsvc.WaitForState(service, state)
 }
 
+// listDependentServices returns a list of names of all services dependent on the given service
+func (m *manager) listDependentServices(serviceHandle windows.Handle) ([]string, error) {
+	// Borrowing the main steps done here from the golang windows/mgr library's ListServices() function, as the
```

Comment:
Could you add the link (https://cs.opensource.google/go/x/sys/+/refs/tags/v0.1.0:windows/svc/mgr/mgr.go;l=176) within the comment?

Reply:
+1